gulpfile.js

```diff
@@ -30,11 +30,11 @@ gulp.task("templates", function() {
     gulp.src(paths.templates)
         .pipe(changed("build/tmpl", {extension: ".js"}))
         .pipe(handlebars({
-            handlebars: require("handlebars")
+            handlebars: require("handlebars"),
```

Comment:
Hm, these are nice changes, but I have a separate branch which ports live-editor to webpack, which will get rid of a bunch of those code anyway. Can you stash this for a future PR? That'll keep this PR to one feature only, and be better timing after mine is merged.

Reply:
Sure, I can stash. Excited for the merge!